components/admin_console/reset_email_modal/reset_email_modal.tsx

```diff
@@ -99,7 +112,6 @@ export default class ResetEmailModal extends React.PureComponent {
                 onHide={this.doCancel}
                 role='dialog'
                 aria-labelledby='resetEmailModalLabel'
-                data-testid='resetEmailModal'
```

Comment:
Why removing this?

Reply:
Sorry @hahmadia , I just focused on unit testing when refactor it.  I already add data-testid again on reset_email_modal.tsx.